### PR TITLE
chore: use similar color for all number cards

### PR DESCRIPTION
**Pull request by vorasmit**

### CODE REVIEW

File: india_compliance/public/js/custom_number_card.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+let OldNumberCard = frappe.widget.widget_factory.number_card;
```
Comment (sagarvora) [inline india_compliance/public/js/custom_number_card.js]:
```suggestion
const FrappeNumberCard = frappe.widget.widget_factory.number_card;
```

File: india_compliance/public/js/custom_number_card.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class CustomNumberCard extends OldNumberCard {
+	render_number() {
+		if (
+			[
+				"Pending e-Waybill",
+				"Pending e-Invoices",
+				"Invoice Cancelled But Not e-Invoice",
+			].includes(this.card_doc.name) &&
+			this.formatted_number === 0
```
Comment (sagarvora) [inline india_compliance/public/js/custom_number_card.js]:
```suggestion
			!this.formatted_number
```

**Review by sagarvora — COMMENTED** [review]
Just a few minor changes. Rest LGTM.

File: india_compliance/public/js/custom_number_card.js (hunk 2 above)
Comment (vorasmit) [reply india_compliance/public/js/custom_number_card.js]: 0 looks more explicit

File: india_compliance/public/js/custom_number_card.js (hunk 1 above)
Comment (sagarvora) [reply india_compliance/public/js/custom_number_card.js]: use `const`

**Pull request merged**